test/metabase/query_processor/streaming/common_test.clj

```diff
@@ -16,6 +16,5 @@
               filename-time                (u.date/parse (streaming.common/export-filename-timestamp))
               filename-time-in-report-zone (t/zoned-date-time filename-time test-timezone)]
           ;; Check that the parsed timestamp matches the expected time within a tolerance of 1 second
-          (is (< (abs (- (.toEpochSecond filename-time-in-report-zone)
-                         (.toEpochSecond now-in-report-zone)))
-                 1)))))))
+          (is (> 1
+                 (.getSeconds (t/duration now-in-report-zone filename-time-in-report-zone)))))))))
```

Comment:
So we're sure the one timestamp is always later than the other one?

Where are these timestamps coming from? Because neither the system or database clocks are guaranteed to be monotonic.

Reply:
Damn you're right I forgot `abs`.

Not monotonic - in principle I can put `mt/with-clock` and sync those timers, I don't know why they are not synced tbh.